### PR TITLE
review the notebook 

### PR DESCRIPTION
**Pull request by KevinYanesG**

### CODE REVIEW

**Comment by KevinYanesG** [comment]
Further comments:
Are these comments referring to what the user would ask himself when seeing the output files:
 <New Folder (WHAT IS THIS?)>
?

`This is essentially the contents of the file <insert file here> that octopus writes.
< Some key features of output >` need the <...> bits be completed?

So far we have explored attributes in postopus that provide string-based information (such as the `run.default.info` attribute) or provide data in rows (such as `run.default.scf.convergence`). 
--> I changed that, because xsf or cube files are also based on strings

I'm not a matplotlib expert, but is there maybe a way to decrease the decimals in the title?
image

And I agree that the methane example is boring, if we cannot find anything better, I would leave the `td` part completely

Also, I don't know if we should mention that we cannot read wavefunction files right now

**Comment by KevinYanesG** [comment]
@fangohr @iamashwin99

**Comment by iamashwin99** [comment]
The parts where I added `<question>` are essentially TODOs that I wanted to clarify with others like Heiko and Martin to document better.
Regarding the wave function, I feel it's not necessary to comment about it here, better ask Hans

**Comment by KevinYanesG** [comment]
Are any `<question>`s left open? :D Otherwise, it would be a good idea to clean it up.

**Comment by fangohr** [comment]
Thanks @KevinYanesG . The changes look fine to me (only looked at the source due to a lack of time).

@iamashwin99 - can you review and if you are happy with it merge, or otherwise modify the bits you do not like?

File: Quick_Start.ipynb [hunk 1]
```diff
@@ -301,134 +301,203 @@
    "id": "5bacb2da",
    "metadata": {},
    "source": [
-    "This tells us that this project had only a self-consistent field simulation (`scf`).\n",
-    "\n",
-    "We can take a look some information about this calculation by running:\n"
```
Comment (iamashwin99) [inline Quick_Start.ipynb:306]: Is this line not helpful?
`"We can take a look some information about this calculation by running:`

**Comment by iamashwin99** [comment]
Looks neat!

File: Quick_Start.ipynb [hunk 2]
```diff
@@ -462,9 +531,9 @@
    "id": "1d783e02",
    "metadata": {},
    "source": [
-    "If you want to access the data for all the iterations, you can use `get_all()` or if you only wanted the converged iteration, you can use `get_converged()`.\n",
+    "If you want to access the data for all the iterations, you can use `get_all()` or if you only wanted the converged iteration, you can use `get_converged()`. In this specific case, both calls would deliver the same output, since in the benzene `inp` file the `OutputDuringSCF` parameter is not set to `True`, so that we only can access to the last converged iteration. For these last two methods, the user doesn't need to provide a `steps` method.\n",
```
Comment (iamashwin99) [inline Quick_Start.ipynb:534]: Shouldnt we specify the source in the previous cell?

Im getting the error 

> ValueError: There is more than one available source for the selected file.

When I run the previous cell.

```
density_data = run.default.scf.density.get(steps=1)
density_data
```

**Pull request merged**